README.md

```diff
@@ -16,9 +16,37 @@ The models can all be used in the same way, using `fit()` and `predict()` functi
 similar to scikit-learn. The library also makes it easy to backtest models,
 and combine the predictions of several models and external regressors.
 
+## Table of Content
+
+- [Darts: Easy manipulation and forecasting of time series.](#darts-easy-manipulation-and-forecasting-of-time-series)
+  - [Table of Content](#table-of-content)
+  - [Install](#install)
```

Comment:
Could we just have the quick install at the top of the README, and leave the step-by-step install at the bottom? This way we can put example usage right after.

Reply:
We can, but then we need to make sure it's also in sphinx docs

Reply:
Good point I was not testing sphinx docs before. The ToC is not working there cause anchors in the .md dont stay when converting to .rst, I will try to work a fix

Reply:
removed ToC